source/infrastructure/dev-tools/index.html.md.erb

```diff
@@ -91,3 +91,26 @@ Run `rvm install ruby 2.7.4`
 Run `rvm use 2.7.4`
 9. install yarn
 Run `brew install yarn`
+
+## Asdf
+
+### Introduction
+
+[Asdf](https://asdf-vm.com) is a version manager used for manging software you install on your computer, it’s a command line interface
+tool that handles installing multiple versions of software packages, allowing you to switch between those versions at runtime.
```

Comment:
Maybe give examples of Ruby and Terraform as they are also mentioned on this page?